src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -9,9 +9,11 @@ import { AuctionArtworksRailPlaceholder } from "../AuctionArtworksRailPlaceholde
 import { tabTypeToContextModuleMap } from "../../Utils/tabTypeToContextModuleMap"
 import { useTracking } from "react-tracking"
 import {
-  clickedArtworkGroupHeader,
-  ClickedArtworkGroupHeaderArgs,
+  ActionType,
+  ClickedArtworkGroup,
```

Comment:
Is it ok that the helper types are still here? This is pretty confusing

Reply:
That's the schema type. small `c` was the helper. The schema types make sense to be there, to verify we send the right things.